### PR TITLE
Cs 657 publish emailaddresschanged event

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

File: src/SFA.DAS.LoginService.Web/Infrastructure/LoginUserClaimsPrincipalFactory.cs [hunk 1]
```diff
@@ -19,7 +19,8 @@ protected override async Task<ClaimsIdentity> GenerateClaimsAsync(LoginUser user
             var identity = await base.GenerateClaimsAsync(user);
             identity.AddClaim(new Claim("family_name", user.FamilyName));
             identity.AddClaim(new Claim("given_name", user.GivenName));
-            identity.AddClaim(new Claim("registration_id", user.RegistrationId.ToString()));
+            identity.AddClaim(new Claim("registration_id", user.ApprenticeId.ToString()));
```
Comment (benagain) [inline src/SFA.DAS.LoginService.Web/Infrastructure/LoginUserClaimsPrincipalFactory.cs:22]: We may as well remove `registration_id` - it's not necessary and may be confusing in the future.

Comment (Pauljgraham) [reply src/SFA.DAS.LoginService.Web/Infrastructure/LoginUserClaimsPrincipalFactory.cs:22]: not until we fix it in ADC and the portal

Comment (Pauljgraham) [reply src/SFA.DAS.LoginService.Web/Infrastructure/LoginUserClaimsPrincipalFactory.cs:22]: ok as long as you're confident the code on ADC will handle it

Comment (Pauljgraham) [reply src/SFA.DAS.LoginService.Web/Infrastructure/LoginUserClaimsPrincipalFactory.cs]: removed

**Review by benagain — APPROVED** [review]

**Pull request closed without merging**